about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link href="style.css" rel="stylesheet">
+      <title>RE Global Summit - About</title>
+  </head>
+  <body>
+    <header>
+      <div class="dark-container">
+        <ul class="dark-ul">
+          <li class="dark-li"><a href="#"><img src="src/social/facebook.svg" alt="facebook"></a></li>
+          <li class="dark-li"><a href="#"><img src="src/social/twitter.svg" alt="twitter"></a></li>
+          <li class="dark-li gray lato"><nav><a href="#">Logout</a></nav></li>
+          <li class="dark-li gray lato"><nav><a href="#">Mypage</a></nav></li>
+          <li class="dark-li gray lato"><nav><a href="#">English</a></nav></li>
+      </ul>
+      </div>
+      <div class="white-container">
+        <a href="index.html"><img class="navbar-logo" src="src/icons/logo.png" alt="logo"></a>
+        <ul class="white-ul">
+          <li class="white-li lato black"><nav><a href="about.html">About</a></nav></li>
+          <li class="white-li lato black"><nav><a href="#">Program</a></nav></li>
+          <li class="white-li lato black"><nav><a href="#">Join</a></nav></li>
+          <li class="white-li lato black"><nav><a href="#">Sponsor</a></nav></li>
+          <li class="white-li lato black"><nav><a href="#">News</a></nav></li>
+          <li class="white-li lato black border-o"><nav><a href="#">RE Campaing</a></nav></li>
+        </ul>
```

Comment:
Hello @ryxtor, your project looks amazing and I would like to thank you for the great job you have done so far 👍🏻! However, you still have a few issues to look at so that It becomes perfect.

- [ ] I have noticed that you are using the `<nav>` tag in between `<li>` tags. I can only term this as misuse. The `nav` tag is meant to be used to enclose a block of that contains navigation links, which is not the case here. I therefore suggest that you remove them `<li class="white-li lato black"><nav><a href="#">Sponsor</a></nav></li>` to `          <li class="white-li lato black"><a href="#">Sponsor</a></li>`